src/tests/radmin/config/control-socket.conf

```diff
@@ -7,9 +7,9 @@
 #  Minimal radiusd.conf for testing radmin
 #
 raddb = raddb
-testdir = build/tests/radmin
-pidfile = ${testdir}/radiusd.pid
-panic_action = "gdb -batch -x src/tests/panic.gdb %e %p > ${testdir}/gdb.log 2>&1; cat ${testdir}/gdb.log"
+builddir = build/tests/radmin
```

Comment:
why rename this?

Reply:
Just to keep the same name used in the others conf files.